hooks/commit-msg.js

```diff
@@ -31,7 +31,9 @@ const commitMessage = fs.readFileSync(commitMessageFilename, {
 });
 
 function commitHasIssue(commitMessage) {
-  const urlIssueRegex = new RegExp(urlBase + issueRegex.source);
+  const urlIssueRegex = new RegExp(
+    urlBase.replace(/\./g, '\\.') + issueRegex.source
```

Comment:
Still a scan problem here it seems ?

Reply:
Hm yes, I forgot to push. In the end, I stop constructing a regexp using strings 😅